public/src/components/channelManagement/helpers/shared.ts

```diff
@@ -14,7 +14,112 @@ export interface Test {
   variants: Variant[];
 }
 
-export type EpicType = 'ARTICLE' | 'LIVEBLOG' | 'APPLE_NEWS' | 'AMP';
+export type EpicType = 'ARTICLE' | 'ARTICLE_HOLDBACK' | 'LIVEBLOG' | 'APPLE_NEWS' | 'AMP';
+
+export interface EpicEditorConfig {
```

Comment:
I had no idea there'd be so many settings, but this is great!

Reply:
Yeah me neither haha - turns out we have a lot of customization so far!